sw/device/lib/testing/meson.build

```diff
@@ -76,3 +76,18 @@ test('mock_mmio_test', executable(
   native: true,
   cpp_args: ['-DMOCK_MMIO'],
 ))
+
+test('mock_dif_uart_test', executable(
+  'mock_dif_uart_test',
+  sources: [
+    meson.source_root() / 'sw/device/lib/testing/dif/mock_dif_uart_test.cc',
+    meson.source_root() / 'sw/device/lib/dif/dif_uart.c',
+    [hw_ip_uart_reg_h],
+  ],
+  dependencies: [
+    sw_lib_testing_gtest,
+    sw_lib_testing_mock_mmio,
+  ],
+  native: true,
+  cpp_args: ['-DMOCK_MMIO'],
+))
```

Comment:
This target definition should live in the same file as the `_test.cc` file.

Reply:
NOTE: haven't addressed yet.

Reply:
We think that the best place for these tests to live is in `sw/device/tests`.